meta/templates/releases/release_template.md

```diff
@@ -0,0 +1,38 @@
+Coming from [opensearch-build__REPLACE_ISSUE_NUMBER__](https://github.com/opensearch-project/opensearch-build/issues/__REPLACE_ISSUE_NUMBER__), release version 1.2. Please follow the following checklist.
+
+### Preparation
+
+- [ ] Assign this issue to a release owner.
+- [ ] Finalize scope and feature set and update [the Public Roadmap](https://github.com/orgs/opensearch-project/projects/1).
+- [ ] Create, update, triage and label all features and issues targeted for this release with `v__REPLACE_MAJOR_MINOR_PATCH__`.
+
+### CI/CD - _Ends Nov 2nd_
```

Comment:
Missed something that needs a replaced annotation, looks like at the top of the issue too.